manifests/n/Nlitesoftd/o/o/NTLite/2024.4.9880.0/Nlitesoftd.o.o.NTLite.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Nlitesoftd.o.o.NTLite
+PackageVersion: 2024.4.9880.0
+PackageLocale: en-US
+Publisher: Nlitesoft d.o.o.
+PackageName: NTLite
+License: none
+Copyright: © 2014-2024 Nlitesoft d.o.o.
+ShortDescription: NTLite Setup

```

Hi @Jvr2022, 

Could the ShortDescription have more details? This line was taken from a guide for the software, so feel free to use a better description, if one is available. 

```suggestion
ShortDescription: NTLite is the best program for making an ISO for a Windows Environment.
```

---

I am gonna change it